repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/Authorisation.java

```diff
@@ -121,4 +124,25 @@ public String getCrudAsString() {
     public void setCrudAsString(final String crudAsString) {
         this.crudAsString = crudAsString;
     }
+
+    @Transient
+    public boolean hasLessAccessThan(Authorisation other) {
+        boolean hasMoreAccessOnCreate = secondHasMoreAccess(this.getCreate(), other.getCreate());
```

Comment:
Not sure about `secondHasMoreAccess ` method name.

Reply:
Can we change method name `secondHasMoreAccess` to `childHasHigherAccess`?

Reply:
agreed